test-suite/js/src/test/scala/org/scalajs/testsuite/utils/Platform.scala

```diff
@@ -24,4 +24,21 @@ object Platform {
 
   def areTypedArraysSupported: Boolean =
     runtime.Bits.areTypedArraysSupported
+
+  def executingInRhino: Boolean = sysProp("rhino")
+  def executingInNodeJS: Boolean = sysProp("nodejs")
+  def executingInPhantomJS: Boolean = sysProp("phantomjs")
+  def typedArrays: Boolean = sysProp("typedarray")
+
+  def isInFastOpt: Boolean = sysProp("fastopt-stage")
+  def isInFullOpt: Boolean = sysProp("fullopt-stage")
+  def isInStrongMode: Boolean = sysProp("strong-mode")
+  def isInProductionMode: Boolean = sysProp("strong-mode")
```

Comment:
Oops. `isInStrongMode` should not be there + `isInProductionMode` is testing `"strong-mode"`.